java/client/src/main/java/glide/api/RedisClient.java

```diff
@@ -78,4 +87,118 @@ public CompletableFuture<Object> customCommand(String[] args) {
                 Command.builder().requestType(Command.RequestType.CUSTOM_COMMAND).arguments(args).build();
         return commandManager.submitNewCommand(command, BaseClient::handleObjectResponse);
     }
+
+    /**
+     * Ping the Redis server.
```

Comment:
It doesn't ping actually.
Copy command description from docs for all commands (https://redis.io/commands/ping/)